StoreApp.BLL/Services/UserService.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections.Generic;
+using AutoMapper;
+using StoreApp.BLL.DTO;
+using StoreApp.BLL.Interfaces;
+using StoreApp.DAL.Entities;
+using StoreApp.DAL.Intefaces;
+
+namespace StoreApp.BLL.Services
+{
+    class UserService : IUserService
+    {
+        IUnitOfWork Database;
+        public UserService(IUnitOfWork uof)
+        {
+            Database = uof;
+        }
```

Comment:
The methods must be splitted by using indents

Reply:
fixed